### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by makermelissa**
It's a little rough around the edges, but it works for a lot of the basic stuff. This is running the ILI9341 displayio driver example without modifications:
IMG_2003

Lots more stuff to add, but it's a good starting point.

### CODE REVIEW

**Comment by makermelissa** [comment]
Please ignore the fact that it's HUGE. It needs to be split into multiple files really, but I wanted to get something working committed.

**Comment by makermelissa** [comment]
Text artifacts fixed.
IMG_2006

**Comment by makermelissa** [comment]
Much better. This is what the final rendering is supposed to look like:
IMG_2008

**Comment by ladyada** [comment]
yay fonts!

**Comment by FoamyGuy** [comment]
@makermelissa Do you think this could work on this older Pi 2.8" TFT: https://www.adafruit.com/product/1601

It does seem to be using ILI9341 driver so I think it maybe it could be used with this. I found the example code in the ILI9341 driver repo and tried it out but I am only getting a blank white screen with no error message. 

I think this example has the pins meant to be used on Feather devices. Did you change the pins used to something different for the Raspberry pi?

**Comment by makermelissa** [comment]
It should work with that. You’ll need some displayio libraries which aren’t on pypi. Yeah, I forgot I changed pins.

**Comment by FoamyGuy** [comment]
I got the `adafruit_display_text` and `adafruit_ili9341` and I think it's all loaded correctly for the example to use them. 

Can you share which pins you used for your setup?

**Comment by makermelissa** [comment]
D25 for d/c D24 for reset and CE0 for chip select if I recall.

**Comment by FoamyGuy** [comment]
Thank you!

Got it working successfully with Raspberry Pi 3 B+

blinka_displayio

**Comment by makermelissa** [comment]
Thanks for testing @FoamyGuy

**Comment by FoamyGuy** [comment]
You're welcome. 

Thank you for working on it. This is very neat! 

`adafruit_display_shapes` seems to be working as well: 

blinka_displayio_shapes

**Comment by makermelissa** [comment]
Oh neat! I wasn't even expecting those to work yet.

**Comment by makermelissa** [comment]
This PR fixes https://github.com/adafruit/Adafruit_Blinka/issues/109. I will create issues for any missing features so that this library can be improved over time.

**Comment by ladyada** [comment]
i haven't had time to test but others have :) so this is good by me!

File: displayio/epaperdisplay.py [hunk 1]
```diff
@@ -0,0 +1,880 @@
...
+        """Display Height"""
+        pass
+
+    @property
+    def bus(self):
+        """Current Display Bus"""
+        pass
+
+
+class FourWire:
```
Comment (tannewt) [inline displayio/epaperdisplay.py:143]: Probably worth deleting these extra stubs.

Comment (makermelissa) [reply displayio/epaperdisplay.py:143]: Oops, yeah. Good catch.

**Comment by makermelissa** [comment]
Thanks

**Pull request merged**